packages/mds-ingest-service/tests/index.spec.ts

```diff
@@ -754,6 +752,39 @@ describe('Ingest Service Tests', () => {
     })
   })
 
+  describe('Tests writeEvent service method', () => {
+    /**
+     * Clear DB after each test runs, and after the file is finished. No side-effects for you.
+     */
+    beforeEach(async () => {
+      await IngestRepository.deleteAll()
+    })
+
+    it('Tests writing an event w/ telemetry for a device that exists', async () => {
```

Comment:
Is there a way to test that a telemetry and an event are both written to their respective tables?

Reply:
A very good idea to actually test the functionality, not that it just doesn't throw an error, yes indeed! Will add tomorrow.